cli/cli/commands/run/run.go

```diff
@@ -562,9 +563,9 @@ func getOrCreateEnclaveContext(
 	var enclaveContext *enclaves.EnclaveContext
 	var err error
 	if isProduction {
-		enclaveContext, err = kurtosisContext.CreateProductionEnclave(ctx, enclaveIdentifierOrName)
+		enclaveContext, err = kurtosisContext.CreateProductionEnclave(ctx, enclaveIdentifierOrName, defaults.DefaultEnableDebugMode)
 	} else {
-		enclaveContext, err = kurtosisContext.CreateEnclave(ctx, enclaveIdentifierOrName)
+		enclaveContext, err = kurtosisContext.CreateEnclave(ctx, enclaveIdentifierOrName, defaults.DefaultEnableDebugMode)
```

Comment:
does this mean anyone using the SDK has to now pass this? Making this a breaking change.

I wonder if we can make it CreateEnclaveWithDebugEnabled and  CreateProductionEnclaveWithDebugenabled; can do this later too. Maybe we can add a `EnclaveCreationOptions` or something.